.github/workflows/update-other.yml

```diff
@@ -11,7 +11,7 @@ permissions:
 jobs:
   update:
     name: Trigger updates for nixvim's stable branches
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04-arm
```

Comment:
Why running this on ARM?

Reply:
Anecdotally, nixpkgs seem to have found most GitHub workflow run faster and/or more efficiently on ARM.

They've migrated many of their workflow jobs to ARM over recent months.

Since we aren't doing anything that requires x86 here, I figured why not.